test.py

```diff
@@ -83,6 +110,37 @@
                 ]
             ),
         )
+    elif cmd_args.segmentation:
+        basic_tfs = [
+            a.Resize(args.inference_resolution, args.inference_resolution,),
+            a.RandomCrop(args.train_resolution, args.train_resolution),
+            a.ToFloat(max_value=255.0),
+        ]
+        val_trans = a.Compose(
+            [
+                *basic_tfs,
+                a.Normalize(mean, std, max_pixel_value=1.0),
+                a.Lambda(
```

Comment:
is this where "lambda is incompatible with multiprocessing" comes from?

Reply:
Yes there're a couple of lambdas in the data-loading, I think I created an issue for that (if not I'll create one).